lighthouse-core/lib/i18n/i18n.js

```diff
@@ -130,10 +130,66 @@ function lookupLocale(locale) {
  * @param {Record<string, string | number>} [values]
  */
 function _preprocessMessageValues(icuMessage, values = {}) {
-  const clonedValues = JSON.parse(JSON.stringify(values));
+  let clonedValues = JSON.parse(JSON.stringify(values));
```

Comment:
rather than mutate this, just `return _processParsedElements(...);` and don't set it to the save var